youtube_dl/extractor/bilibili.py

```diff
@@ -497,11 +497,10 @@ def _real_extract(self, url):
                 'id':       video_id + re.search('([^/?#&]+).flv', chunk['url']).group()[-24:-5],
                 'duration': float_or_none(chunk.get('length'), 1000),
                 'formats':  formats,
-                **info
             })
 
-        return {
-            '_type':       'multi_video',
-            'entries':     entries,
-            **info
-        }
+        for e in entries:
+            e.update(info)
```

Comment:
What's the purpose here ?

Reply:
purpose of this change: **info is not supported by the old python versions

purpose of the code: the entries are segments of the recorded video, so I use the same title (and other information) for them